server/db/queries/Tests.js

```diff
@@ -0,0 +1,23 @@
+const pages = require('./Pages');
+
+
+module.exports = {
+  // get all results in DB
+  getAllResults: () => 'SELECT * FROM tests',
+
+  // get results for test
+  getTest: (testName) =>
+    `SELECT * FROM tests WHERE name = '${testName}'`,
+
+  // create new test for a page
+  insertNewTest: (testName, pageName) =>
+    `INSERT INTO tests (name, result_a, result_b, page_id) VALUES ('${testName}', 0, 0, (${pages.getPageID(pageName)}))`,
+
+  // increment result_a of a test
+  incrementTest: (testName) =>
+    `UPDATE tests SET result_a = result_a + 1 WHERE name = '${testName}'`,
+
+  // increment result_a of a test
```

Comment:
This comment is the same as comment in line 16. They're supposed to be different, right?